deploy_tools/settings_local.py

```diff
@@ -10,7 +10,7 @@
 ADMIN_EMAILS = ['your_email@gmail.com',]
 
 # Custom Local Variables
-SERVER_HOST = 'localhost:7777'
+SERVER_HOST = 'daya.ischool.berkeley.edu:7777'
```

Comment:
Reverting this. Make sure you change the settings_local.py in apps directory, not the deploy_tools one.